wazo_chatd/plugins/presences/bus_consume.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, dao, notifier):
         self._notifier = notifier
 
     def subscribe(self, bus_consumer):
-        bus_consumer.on_event('auth_tenant_created', self._tenant_created)
```

Comment:
That's sad. Did you create a ticket to eventually fix the event name and deprecate this one?

Reply:
https://wazo-dev.atlassian.net/browse/WAZO-1669